networkx/relabel.py

```diff
@@ -4,7 +4,9 @@
 
 
 def relabel_nodes(G, mapping, copy=True):
-    """Relabel the nodes of the graph G according to a given mapping.
+    """Relabel the nodes of the graph G according to a given mapping. Note
+    that the original node ordering is preserved (even if mapping is defined
+    partially).
```

Comment:
Keep the first line as it was. Add the note as a paragraph separated from the first line by a blank line.
That's because the Sphinx documentation system uses the first line (up to the first blank line) as a short description of the function when it is referred to in the docs.